app/Fission/Internal/Development.hs

```diff
@@ -132,6 +134,7 @@ mkConfig dbPool processCtx httpManager logFunc = Config {..}
     awsSecretKey  = "SOME_AWS_SECRET_KEY"
     awsZoneID     = "SOME_AWS_ZONE_ID"
     awsDomainName = "SOME_AWS_DOMAIN_NAME"
+    awsMockEnabled = AWS.MockEnabled True
```

Comment:
This is mostly getting me familiar with the code, so excuse my ignorance 😅 

This means that the mock is enable by default in dev, right?
Do we even need an environment variable then, or we do we need mocking in non-test envs as well?
Or is the reasoning more like, why not have an env variable if it's easy to setup?

Also, thinking of testing, do we want mocking to be enabled per external service, or is like a "global" switch good enough?

Reply:
@icidasset you're absolutely right that this is a code smell 💯

The "right" way to do this is to write a `class MonadAWS`, and use that to do the switching — either with a flag or some `Maybe`s, but all isolated into the top-level `instance` declaration.

We're doing this for now just to get unblocked for local dev. I don't have the time this week to teach that style, so we're going to circle back. I've been eliminating `RIO` pretty aggressively in the DB adapter switch.

Reply:
so @icidasset this function here `mkConfig` is a helper for if you are trying to run code in the repl taht requires config.

So essentially this turns on mocking when you are tinkinging in the repl

I think we will want a per service mock for the most part as we expand? Given that I can rename this to `route53Mock`. what are your thoughts?

Reply:
@bechurch Ah gotcha. For some reason I keep thinking that this `Development.hs` file is for the dev environment. Probably because I'm so used Phoenix's config files 🤔 Totally makes sense then 👍 

Yeah, per service mocks in dev makes sense (to enable mocking, or not, totally depends on the service I imagine). Also regarding testing, I guess in the tests we'll always have mocks on by default?